torch/testing/_internal/common_methods_invocations.py

```diff
@@ -63,6 +63,10 @@ def __init__(self,
                  dtypesIfCUDA=None,  # dtypes this function is expected to work with on CUDA
                  dtypesIfROCM=None,  # dtypes this function is expected to work with on ROCM
                  test_inplace_grad=True,  # whether to gradcheck and gradgradcheck the inplace variant
+                 is_autodiffed=False,  # if a op's aten::node is expected to be symbolically autodiffed
```

Comment:
Can I accidentally support autodiff?

Reply:
This can be addressed by improving the error message on the stacked PR, see it for a comment